plugins/README.md

```diff
@@ -0,0 +1,49 @@
+﻿# Plugins
+
+> **IMPORTANT:** This sample is for educational purposes only and is not recommended for production deployments.
+
+Plugins are cool! They allow Chat Copilot to talk to the internet. Read more about plugins here [Understanding AI plugins in Semantic Kernel](https://learn.microsoft.com/en-us/semantic-kernel/ai-orchestration/plugins/?tabs=Csharp) and here [ChatGPT Plugins](https://platform.openai.com/docs/plugins/introduction).
+
+## Available Plugins
+
+> These plugins in this project can be optionally deployed with the Chat Copilot [WebApi](../webapi/README.md)
+
+- [WebSearcher](./web-searcher/README.md): A plugin that allows the chat bot to perform Bing search.
+- More to come. Stay tuned!
+
+## Third Party plugins
+
+You can also configure Chat Copilot to use third party plugins.
+
+> All no-auth plugins will be supported.
+
+> All service-level-auth plugins will be supported.
+
+> OAuth plugins will NOT be supported.
+
+Read more about plugin authentication here: [Plugin authentication](https://platform.openai.com/docs/plugins/authentication)
+
+## Plugin Configuration in Chat Copilot
+
+### Prerequisites
+
+1. The name of your plugin. This should be identical to the `NameForHuman` in your plugin manifest.
+2. Url of your plugin.
```

Comment:
Specify whether this should be the manifest url or the url pointing to the openapi spec. 

It would also be good to all out the requirements for plugin name and other fields defined by open ai here: https://platform.openai.com/docs/plugins/getting-started/plugin-manifest